src/modules/customer/containers/HappyUserPage/components/WizardForm/WizardFormPages/EighthPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form/immutable';
+import { FormattedMessage } from 'react-intl';
+
+import messages from '../messages';
+import formConstants from '../constants';
+
+const WizardFormEighthPage = props => {
+  const { handleSubmit, previousPage } = props;
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className="wizard-form__container">
+        <label className="wizard-form__label">
```

Comment:
All labels should set focus to the corresponding input fields. For this the inputs should get ids and labels should be referenced this id.